testkit/src/main/java/org/apache/calcite/test/SqlOperatorTest.java

```diff
@@ -3258,8 +3264,10 @@ static void checkOverlaps(OverlapChecker c) {
   }
 
   @Test void testMinusOperator() {
-    final SqlOperatorFixture f = fixture();
-    f.setFor(SqlStdOperatorTable.MINUS, VmName.EXPAND);
+    SqlOperatorFixture f = fixture()
```

Comment:
I'm not sure if it only affects these operators, but it looks like it can be expanded later and the problem is not big